ci/Jenkinsfiles/build.groovy

```diff
@@ -29,21 +29,60 @@ String getCommitSha1() {
   return sh(returnStdout: true, script: 'git rev-parse HEAD').trim();
 }
 
+void dockerPull(String image) {
+  sh "docker pull ${image}"
+}
+
+void dockerTag(String image, String tag) {
+  sh "docker tag ${image} ${tag}"
+}
+
+void dockerPush(String image) {
+  sh "docker push ${image}"
+}
+
+void dockerDeploy(String imageName) {
+  String imageTag = "${ORG}/${imageName}:${VERSION}"
+  String internalImage = "${DOCKER_REGISTRY}/${imageTag}"
+  String explorerImage = "${DOCKER_REGISTRY}/${imageTag}"
```

Comment:
The registry to which you push should be a dedicated one: `${NUXEO_DOCKER_REGISTRY}` for instance, to be defined in the `environment` section.

The `DOCKER_REGISTRY` environment variable is set in Jenkins by the environment and refers to the Docker registry hosted on the [internal Nexus in Jenkins X](https://nexus.platform.dev.nuxeo.com/), see https://nexus.platform.dev.nuxeo.com/#browse/browse:docker-registry.